show/main.py

```diff
@@ -1383,8 +1388,20 @@ def runningconfiguration():
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def all(verbose):
     """Show full running configuration"""
-    cmd = "sonic-cfggen -d --print-data"
-    run_command(cmd, display_cmd=verbose)
+    cmd = ['sonic-cfggen', '-d', '--print-data']
+    stdout = get_cmd_output(cmd)
+
+    try:
+        output = json.loads(stdout)
+    except JSONDecodeError as e:
+        click.echo("Failed to load output '{}':{}".format(cmd, e))
+        raise click.Abort()
+
+    if 'bgpraw' in output or not multi_asic.is_multi_asic():
+        bgpraw_cmd = [constants.RVTYSH_COMMAND, '-c', 'show running-config']
+        bgpraw = get_cmd_output(bgpraw_cmd)
```

Comment:
>bgpraw [](http://example.com/codeflow?start=8&length=6)

return `(stdout, rc)`. After calling this function, check `rc` first. If rc non zero, output['bgpraw'] should be empty string.

Reply:
Same as above, it will be empty string by default if rc is non-zero.